Sources/SwifterSwift/MapKit/MKMapViewExtensions.swift

```diff
@@ -9,6 +9,9 @@
 #if canImport(MapKit)
 import MapKit
 
+#if canImport(UIKit)
```

Comment:
This will completely block macOS. Instead what we need is a `typealias` to use in the function parameter. Try:
```Swift
#if canImport(UIKit)
import UIKit
private typealias EdgeInsets = UIEdgeInsets
#elseif os(macOS)
import Foundation
private typealias EdgeInsets = NSEdgeInsets
#endif
```
I've made the aliases `private` here because it's not the right place to declare them publicly.

Reply:
@guykogus EdgeInsets public olmadıkça hata alıyorum.

Reply:
@guykogus I get an error unless EdgeInsets is public.

Reply:
Of course, otherwise the function can't be publicly visible 🤦‍♂
In that case can you make a new file in the `Shared` folder, alongside `ColorExtensions.swift`, called `EdgeInsetsExtensions.swift` and put the declaration in there? Technically we could then move the extensions from `UIEdgeInsetsExtensions.swift` but let's save that for later 😄

Reply:
@guykogus Wouldn't it be more accurate to include it directly in UIEdgeInsets Extensions.swift 🤔? Shall I do this?

Reply:
@mustafagunes no because we're creating an extension for both `UIEdgeInsets` and `NSEdgeInsets` together, so it belongs in a unique combined extension file, just like `Color.swift` does for `UIColor` and `NSColor`.